ext/oss/sink.go

```diff
@@ -22,6 +22,10 @@ import (
 	"github.com/pkg/errors"
 )
 
+const (
+	MAX_TEMP_FILE_RECORD_NUMBER = 50000
```

Comment:
let's make this configurable? 0 means no partial upload

Reply:
I was thinking to hide this from user configuration since this logic is solely used for oss sink internal logic. But we can have it configurable

Reply:
sure, that’s also fine. please take a note if the source generates records that contains non uniform fields, this partial upload won’t work well on csv and tsv